api/filter/http/gzip.proto

```diff
@@ -50,29 +50,24 @@ message Gzip {
 
   // Set of strings that allows specifying which mime-types yield compression; e.g.
   // application/json, text/html, etc. When this field is not specified, compression will be applied
-  // to the following MIME-TYPES: html, text, css, js, json, svg, xml. xhtml".
+  // to the following MIME-TYPES: html, text, css, js, json, svg, xml, xhtml.
   repeated string content_type = 6 [(validate.rules).repeated = {min_items: 0, max_items: 50}];
 
-  // Value that disables compression if response contains Etag (entity tag)
-  // header. This option is disabled by default (false), which means that strong Etags will be
-  // re-writen to its weak form before being dispatched to the client. If for some reason the client
-  // does not support weak validation, it is recommended to have this enabled.
-  google.protobuf.BoolValue disable_on_etag = 7;
+  // Value that disables compression if response contains Etag (entity tag) header. This option is
+  // not enabled by default (false), which means that weak tags will be preserved and strong Etags
```

Comment:
Everything is default to 0, false, "" in proto3. There is no need to explain it for every field. That is part of design goal of proto3.